stage0/src/fw_cfg.rs

```diff
@@ -34,6 +34,9 @@ const FWCFG_PORT_DMA: u16 = 0x514;
 
 const SIGNATURE: &[u8] = b"QEMU";
 
+// Bit 1 indicates that DMA access is enabled on the fw_cfg device.
+const DMA_ENABLED: u8 = 1 << 1;
```

Comment:
We have the `bitflags` crate in stage0, maybe declare a proper struct for the feature flags?

Reply:
Done.